src/rai_core/rai/agents/spatiotemporal/ros2.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (C) 2025 Robotec.AI
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from typing import Annotated, Dict, List, Optional
+
+from langchain_core.messages import BaseMessage
+
+from rai.agents.spatiotemporal.spatiotemporal_agent import (
+    Header,
+    Point,
+    Pose,
+    PoseStamped,
+    Quaternion,
+    SpatioTemporalAgent,
+    SpatioTemporalConfig,
+)
+from rai.communication import ROS2ARIConnector
+from rai.tools.ros2.topics import GetROS2ImageTool
+
+
+class ROS2SpatioTemporalConfig(SpatioTemporalConfig):
+    camera_topics: List[str]
```

Comment:
This would make it more consistent with the code below
```suggestion
    camera_topics: List[Annotated[str, "camera topic"]]
```

Reply:
63f68bb150062fcf54f10ef5b05e4e4d4e4ee668